config/jobs/kubernetes/sig-node/sig-node-presubmit-cpumanager.yaml

```diff
@@ -0,0 +1,120 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-node-e2e-cpumanager
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-containerd
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    always_run: false
+    max_concurrency: 12
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191212-28f7cbc-master
+        args:
+        - --root=/go/src
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        - --scenario=kubernetes_e2e
+        - -- # end bootstrap args, scenario args below
+        - --deployment=node
+        - --gcp-project=k8s-jkns-pr-node-e2e
+        - --gcp-zone=us-west1-b
+        - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+        - --node-tests=true
+        - --provider=gce
+        - --test_args=--nodes=8 --focus="\[CPUManager\]" --skip="" --flakeAttempts=2
```

Comment:
Ref. https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-dev/NNmEGUsJObg/Vszw5EZJAAAJ and https://github.com/kubernetes/test-infra/pull/15516 you should remove `--flakeAttempts=2`

Reply:
Sure, will do.

Reply:
All the serial node tests should run with `--nodes=1`. `--skip=""` is also set implicitly by default.

Reply:
OK, I've changed the --nodes=1. 

Also, after reviewing a bit more, I think it would be cleaner to just add these new jobs to the existing sig-node-presubmit.yaml file, instead of creating 2 new config files. I'm working on testing that now, will update this PR will all the above changes shortly.

Appreciate all the comments and input! Definitely very helpful to get this cleaned up.